lib/screens/send/send_contact.dart

```diff
@@ -187,6 +171,90 @@ class _SendToContactScreenState extends State<SendToContactScreen> {
     );
   }
 
+  Widget recentTransctions() {
+    List<Widget> listItems = List();
+    final sorted =
```

Comment:
the transactions already supposed to be sorted from the old to new ones. you might